### DISCUSSION TITLE
x y velocity (from stateEstimate) doesn't graph even though I have a flowdeck attached

### DISCUSSION THREAD

**Discussion by ugrm** [bitcraze/discussions#1412] [Q&A] [answered]
I have the flowdeck attached and followed the instructions in the **Crazyflie 101 EPFL-LIS lecture (2023)** just in case. Everything else (pose, etc) graphs properly. However, the x,y velocity from stateEstimate does not plot properly. It stays constant even if I move the drone manually.
What could be the problem? I'm assuming it might have to do something with the flowdeck?
![image](https://github.com/bitcraze/discussions/assets/145661518/649d93f9-1960-41a9-9b4a-59ea0d069dfd)

**Comment by ugrm** [comment, marked as answer]
The flowdeck was messed up. I changed it to a new one and it works fine!

**Reply by knmcguire** [reply]
ah yes, usually it is a sign that the flowdeck is not working. Next time you could check out the console tab to see if there is any error message with initialization.

**Reply by ugrm** [reply]
gotcha! I'll keep that in mind
